runtime/v2/shim/util_unix.go

```diff
@@ -78,6 +78,10 @@ func AnonDialer(address string, timeout time.Duration) (net.Conn, error) {
 	return net.DialTimeout("unix", "\x00"+address, timeout)
 }
 
+func AnonReconnectDialer(address string, timeout time.Duration) (net.Conn, error) {
+	return AnonDialer(address, timeout)
+}
```

Comment:
@jterry75 Is there something smarter I should be doing in the linux reconnect case? Do we know if containerd restart timeouts are a problem in the linux scenario?

Reply:
It's not because they use a fifo